### PR TITLE
integrations: Add screenshots and notes for papertrail.

### PR DESCRIPTION
**Pull request by anirudhjain75**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@anirudhjain75, it looks like you've already signed the Dropbox CLA. Thanks!

**Comment by femnad** [comment]
Could you remove the `.DS_Store` files?

Also, could you add the notes in a text only file format, like .txt or markdown. That way it is easier to keep track of changes.

By the way, you don't have to close this request to edit it. See this document if you need help in fixing your commits: https://github.com/zulip/zulip-gci/blob/master/docs/fixing-commits.md

**Comment by femnad** [comment]
But that `.txt` file is really an RTF file with changed extension, right? Since it still has all those metadata:

> 
> {\rtf1\ansi\ansicpg1252\cocoartf1348\cocoasubrtf170__
> {\fonttbl\f0\fswiss\fcharset0 Helvetica;}
> {\colortbl;\red255\green255\blue255;}
> \paperw11900\paperh16840\margl1440\margr1440\vieww10800\viewh8400\viewkind0
> \pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\pardirnatural

So it really should be a .txt file with only the text you put in it.

File: webhook-integrations/papertrail/notes.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+Papertrail
+    Papertrail only gives sends logs to its integrated tools.
```
Comment (pweaver) [inline webhook-integrations/papertrail/notes.txt]: Extraneous gives?

**Comment by pweaver** [comment]
When downloading the jpegs, they are still broken. Are they a different file extension?

**Pull request closed without merging**